### ISSUE TITLE
What is the setting for SyncBN experiments in Table.7

### ISSUE THREAD

**Issue by VectorYoung** [open-mmlab/mmdetection#1121]
Hi, from Table.7 in the technical report, the SyncBN shows comparable results with FrozenBN when applied on backbone. May I ask what is the seeting for SyncBN? Do you just replace BN with SyncBN and initialize with Imagenet-pretrained weights, and set eval=True and requires_grad=True during training, or just replace BN with SyncBN and training from scratch? Thanks.

**Comment by hellock** [comment]
Exactly the same settings as FrozenBN are used, except for the BN type.

**Comment by VectorYoung** [comment]
Hi @hellock , so do you mean SyncBN uses the same ImageNet-pretrained model weights as FrozenBN?

**Comment by hellock** [comment]
Right.

**Comment by YueshangGu** [comment]
Hi @hellock @VectorYoung Why set all BN or SyncBN to eval mode when training? When use eval, BN or SyncBN will use running mean and running var to compute forward and will not update running mean and running var, and I want to know which mean and var will be used to compute backward, batch mean or running mean? If I want to use SyncBN and train from scratch, what should I set during training?

**Comment by hellock** [comment]
@YueshangGu The batch size is usually small for detection and not enough for BN training. If we use SyncBN we usually set it to train mode.
